### PR TITLE
net-irc/znc-clientbuffer: Add myself as maintainer, fix open bug and QA issue

### PR DESCRIPTION
**Pull request by NexAdn**
This PR adds myself as maintainer for net-irc/znc-clientbuffer and fixes the currently only open bug associated with the package.
Furthermore, a small QA fix (use PV in SRC_URI) was added.

### CODE REVIEW

File: net-irc/znc-clientbuffer/znc-clientbuffer-1.0.48.ebuild [hunk 1]
```diff
@@ -1,11 +1,13 @@
-# Copyright 2020 Gentoo Authors
+# Copyright 2020-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```
Comment (ConiKost) [inline net-irc/znc-clientbuffer/znc-clientbuffer-1.0.48.ebuild:4]: How about bump to EAPI=8?

Comment (NexAdn) [reply net-irc/znc-clientbuffer/znc-clientbuffer-1.0.48.ebuild]: done

File: net-irc/znc-clientbuffer/znc-clientbuffer-1.0.48.ebuild [hunk 2]
```diff
@@ -18,20 +20,18 @@ RDEPEND="${DEPEND}"
 
 DOCS=( README.md )
```
Comment (ConiKost) [inline net-irc/znc-clientbuffer/znc-clientbuffer-1.0.48.ebuild:21]: You can drop this, as it's the default, when not being set.

Comment (NexAdn) [reply net-irc/znc-clientbuffer/znc-clientbuffer-1.0.48.ebuild]: done

**Review by ConiKost — CHANGES_REQUESTED** [review]
lgtm, but please squash into one commit.
@thesamesam Could you merge, since I am devaway now.

**Comment by thesamesam** [comment]
sure, thanks!

**Comment by thesamesam** [comment]
I've squashed but also turned it into a revbump as it's a substantial build system change, just to be less risky.

**Comment by ConiKost** [comment]
Thanks!

**Pull request closed without merging**